CHANGELOG.md

```diff
@@ -4,14 +4,76 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]:
+## [0.6.0]: 0.6.0 - TBD.
+
+TODO:
```

Comment:
We should highlight a few features:
- Prebuilt images with kubernetes to speed up instance auto scaling
- Provide AMIs for tarmak from Jetstack
- Calico kubernetes backend

Worth metioning: 
- Improved SSH libraries
- Host Key no verified through AWS instance tags

Action required:
- Not too sure if we have breaking changes